app/views/home/search.json.jbuilder

```diff
@@ -13,6 +13,7 @@ json.features @parks do |park|
     json.photos park.photos.map {|photo| photo.photo.url}
     json.mini_photos park.photos.map {|photo| photo.photo.mini.url}
     json.thumb_photos park.photos.map {|photo| photo.photo.thumb.url}
-    json.hour_fee park.hour_fee
+    json.hour_fee park.hour_fee # TODO Deprecate そのうち削除
```

Comment:
[rubocop] Annotation keywords should be all upper case, followed by a colon and a space, then a note describing the problem.